isis/python_bindings/apps/findFeaturesSegment/findFeaturesSegment.py

```diff
@@ -311,7 +313,6 @@ def findFeaturesSegment(ui):
     else: 
         nthreads = int(multiprocessing.cpu_count())
 
-
     pool = ThreadPool(ceil(nthreads/len(img_list)))
```

Comment:
Not part of the PR, but I just saw this line.  Is this correct..?  If the user passes in nthreads=8 and there are 8 images, then won't this end up with a threadpool of 1?

Reply:
I feel like there was a reason for that but in hindsight it doesn't make sense 🤔 I'll look into it some more.